README.md

```diff
@@ -23,7 +23,7 @@ pip install apache-airflow-provider-transfers
 
 ## Example DAGs
 
-Checkout the [example_dags](./example_dags) folder for examples of how the UniversalTransfeOperator can be used.
+Checkout the [example_dags](./example_dags) folder for examples of how the Universal Transfer Operator can be used.
```

Comment:
@phanikumv there is another PR that changes the name from `Universal Transfer Operator` to `UniversalTransfeOperator` i think we all should agree on one name.
line: 11 of the readme.md
https://github.com/astronomer/apache-airflow-provider-transfers/pull/43/files

Reply:
I am ok with anything. However the issue here is that the word is spelled wrong - notice the missing "r"  in `UniversalTransfeOperator`

Reply:
Yes, I would prefer UniversalTransferOperator too, and maybe enclose it in code blocks `UniversalTransferOperator ` since we directly want to associate with that operator from the example DAG.

Reply:
Closing in favour of https://github.com/astronomer/apache-airflow-provider-transfers/pull/43 :)